src/test/java/com/wojtek/parkingmeter/helpers/ChargeCalculatorTest.java

```diff
@@ -0,0 +1,124 @@
+package com.wojtek.parkingmeter.helpers;
+
+import com.wojtek.parkingmeter.helpers.enums.TicketType;
+import com.wojtek.parkingmeter.model.TicketEntity;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import java.time.LocalDateTime;
+
+public class ChargeCalculatorTest {
+
+
+    @Test
+    public void RegularLessThanOneHour() {
```

Comment:
Nazwy metod powinny zaczynać się z małej litery. To co jest tutaj, jest niespójne z resztą kodu.

Reply:
Ok, mój błąd i niekonsekwencja.